api-spec/v1/openapi.yaml

```
@@ -62,6 +62,13 @@ paths:
       operationId: authUsers
       summary: Get user information
       description: 'GET user information'
+      parameters:
+        - in: header
+          name: x-rpt-id
+          required: false
+          description: Optional RPTID

```

```suggestion
          description: Optional RPT ID used to track login attempts to payment notices

```
please update other description fields too